.evergreen/download-mongodb.sh

```diff
@@ -56,6 +56,7 @@ get_mongodb_download_url_for ()
    VERSION_MONGOSH="2.1.1"
    # Set VERSION_RAPID to the latest rapid release each quarter.
    VERSION_RAPID="7.3.1"
+   VERSION_80="v8.0-latest"
    VERSION_70="7.0.9"
    VERSION_60_LATEST="v6.0-latest"
```

Comment:
Not relevant for this PR, but do you remember why we keep 6.0-latest around when we don't have 7.0-latest? Is there a test requirement for 6.0-latest?

Reply:
Looking at the history, I suspect we should have removed it when we added VERSION_60 stable.